src/Functions/DateTimeTransforms.h

```diff
@@ -21,6 +21,7 @@
 namespace DB
 {
 
+static constexpr auto nanosecond_multiplier = 1000000000;
```

Comment:
Cosmetic: We can make this slightly more beautiful:

```cpp
static constexpr auto nanosecond_multiplier = 1'000'000'000;
static constexpr auto microsecond_multiplier = 1'000'000;
static constexpr auto millisecond_multiplier = 1'000;
```

Also, could we please go from coarse to fine-granular? Let's use order: `millisecond_multiplier`, `microsecond_multiplier`, `nanosecond_multiplier`.